src/main/java/MemGlobeApplication.java

```diff
@@ -5,16 +5,20 @@ public class MemGlobeApplication {
 
     public static void main(String[] args) {
         ArrayList<Globe.Trivia> currentSet = new ArrayList<>();
-        //get path to file to import
-        String file = "src/main/resources/countriesAndCapitals.txt";
+        //get path of file to import
+        String file = "src/main/resources/countriesAndAreaAndPopulation.txt";
 
         CardSets.importList(file, currentSet);
 
         System.out.println("currentSet has size: " + currentSet.size());
-        System.out.println(currentSet.get(4).id);
-        System.out.println(currentSet.get(4).answers);
-        System.out.println(currentSet.get(11).id);
-        System.out.println(currentSet.get(11).answers);
+
+        for (int i = 0; i<6;i++){
```

Comment:
Why are you limiting your loop to 6 counts? Wouldn't is make more sense to loop through the whole currentSet?